corehq/apps/hqadmin/static/hqadmin/js/faceted_report.js

```diff
@@ -0,0 +1,13 @@
+hqDefine('hqadmin/js/faceted_report', function() {
+    var standardHQReport, asyncHQReport;
```

Comment:
If any code was previously using these vars (probably via extending `faceted_report.html` and then extending the `js-line` block) it's not going to have access to them anymore...and if no code is using these vars, might as well delete them.